Documents/Pseudocode/Pseudocode_Final

```diff
@@ -108,7 +108,7 @@ Class Network
 	Network(const string &previous_network_filename)
 	{
 	    //readIn(previous_network_filename);	
-		//prompt user: do they want to update hyperparameters before training?
+		//prompt user: do they want to update hyperparameters before training? //	why not, read the hyperparameters form the file only if the user wants it? [-Ami]
```

Comment:
Dunno, I thought that maybe we'd be storing data in binary mode (using a struct to make writing data easier, if possible), and if that were the case editing the file using the cmd prompt would be much easier. 

Additionally, the file may end up being a smattering of numbers/moonspeak, so that the user may not know what exactly to edit, or may not want to find it in the file.

Reply:
Or the existing file may have hyperparameters that the user wants to fiddle with, without having to run setup for a whole new network. I thought it might be easy enough to write, and it could be a convenience to the user.